style.css

```diff
@@ -0,0 +1,41 @@
+@media (min-width: 320px) {
+	.wrap__title {
+		font-size: 10px;
+		background-color: darkblue;
+	}
+}
+
+@media (min-width: 481px) {
+	.wrap__title {
+		font-size: 20px;
+		background-color: blue;
+	}
+}
+
+@media (min-width: 641px) {
+	.wrap__title {
+		font-size: 30px;
+		background-color: lightblue;
+	}
+}
+
+@media (min-width: 961px) {
+	.wrap__title {
+		font-size: 40px;
+		background-color: darkred;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/CSS17K/pullRequest?prid=5813015)